app/views/users/index.html.erb

```diff
@@ -1,6 +1,8 @@
 <div class="wrapper">
   <div class="add-user-btn">
+    <% if current_user.is_admin %>
     <%= link_to 'Add User', new_user_path, class: 'btn-primary w-button' %>
```

Comment:
It can be helpful to indent blocks within an `if-else`.

Reply:
This issue was resolved